### ISSUE TITLE
Re run paused jobs

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/adventist-dl#489]
# Summary

While we were trying to meet the SDAPI ingest deadline, Rob paused the ADL jobs triggered by Katharine and her team. He did so with the following examples

```
GoodJob::Job.where(“finished_at is NULL AND serialized_params->>‘tenant’ = ?“, “3929b765-8d25-48ab-986f-3ce4edf75a6b”).update_all(error: ‘SKIP’, finished_at: Time.now)
```

```
GoodJob::Job.where(“finished_at is NULL AND serialized_params->>‘tenant’ = ?“, “3929b765-8d25-48ab-986f-3ce4edf75a6b”).update_all(error: ‘SKIP-sdapi’, finished_at: Time.now)
```

Now we need to re find jobs with error "SKIP" and "SKIP-sdapi" and re spawn them to completion. 

```
adl_jobs = GoodJob::Job.where(error: 'SKIP')
other_jobs = GoodJob::Job.where(error: 'SKIP-sdapi')
```

Use the playbook article listed below, or consult with Rob about the best way to re trigger them if needed. He has office hr on Tuesday 2:30PM PDT.

Hopefully it will resolve the concerns Katharine mentioned here: https://assaydepot.slack.com/archives/C0313NJV9PE/p1688740330826599

# Acceptance Criteria

The previously paused jobs run to completion. 
Verify if the works mentioned in [this slack convo](https://assaydepot.slack.com/archives/C0313NJV9PE/p1688740330826599) get resolved. 

# NOTES
How to run jobs: https://playbook-staging.notch8.com/en/dev/workers/good-job/good_job

```
job = GoodJob::Job.find_by(active_job_id: id)
job.update(performed_at: Time.now) # lock job
active_job_data = job.serialized_params.deep_dup
value = ActiveJob::Base.execute(active_job_data) # this runs the actual job

job.update(finished_at: Time.now, error: nil) # this is for marking it done if it succeeds
```

**Comment by orangewolf** [comment]
all previously set aside jobs have been put back in the queue:
```
job = GoodJob::Job.where(error: 'SKIP').update_all(error: nil, finished_at: nil)
job = GoodJob::Job.where(error: 'SKIP-sdapi').update_all(error: nil, finished_at: nil)
```

**Comment by ShanaLMoore** [comment]
Marking blocked until the jobs finish processing.

**Comment by ShanaLMoore** [comment]
Jobs completed. TODO: ask @KatharineV to review the concerns to see if they've been resolved.

**Comment by ShanaLMoore** [comment]
Per [Katharine](https://assaydepot.slack.com/archives/C0313NJV9PE/p1689611025243569?thread_ts=1688740330.826599&cid=C0313NJV9PE): 

Thanks for the update! Some of the works that had issues did resolve themselves. The examples I gave above are now showing in the viewer. Thumbnails are not showing, and some other works have not rendered in the viewer. (examples that are not in viewer https://adl.b2.adventistdigitallibrary.org/concern/generic_works/c305_b002_f01_box_2_fld_1_jan_to_feb_1921?locale=en and https://adl.b2.adventistdigitallibrary.org/concern/generic_works/c305_b002_f02_box_2_fld_2_mar_to_apr_1921?locale=en)
In summary, a good amount of the remaining jobs did finish, but some works are still unfinished. We have lots of works that need to be processed for the viewer (that were uploaded before the print gem was installed), so I won't worry about these if you believe they will be fixed when we fix everything that remains.
